### ISSUE TITLE
perplexing log message - gene vs. genotype

### ISSUE THREAD

**Issue by mah11** [pombase/pombase-chado#764]
What do these mean?

    warning in bf037075feb20f28: some data from annotation isn't used: genotype
    warning in bf037075feb20f28: phenotype annotation for "normal RNA level during cellular response to replete zinc (FYPO:0006830)" must have genotype information has gene instead

They seem to go together but I don't understand how Canto could stick a phenotype on a gene ....

**Comment by kimrutherford** [comment]
> They seem to go together but I don't understand how Canto could stick a phenotype on a gene ....

Yep, that's very bad.  :-)

The session with the problem is rather old (2014-06-09) so I'm hoping that this was caused by an old bug that's not there anymore.  I checked in the database for any similar problems.  Unfortunately there are quite a few but they are all old sessions (mostly 2012) that were just created for testing.  None are approved.

The dodgy annotation in bf037075feb20f28 is

     SPAC25B8.19c/loz1     FYPO:0006830   Reporter gene assay  + Zn ions  assayed_using zrt1

with comment: "fig1"

It doesn't get displayed when you look at the session at curation.pombase.org.

There is a non-dodgy annotation in the session that looks very similar:

     loz1+[Overexpression]   FYPO:0006830   Reporter gene assay  + Zn ions  assayed_using zrt1 


I've fixed the session by removing the broken annotation.
